plugin/trino-pinot/pom.xml

```diff
@@ -14,15 +14,32 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
-        <dep.pinot.version>0.12.1</dep.pinot.version>
+        <dep.pinot.version>1.1.0</dep.pinot.version>
+        <!--
+           Project's default for air.test.parallel is 'methods'. By design, 'instances' makes TestNG run tests from one class in a single thread.
```

Comment:
I think Pinot connector already migrated to JUnit from TestNG. https://github.com/trinodb/trino/commit/8025f2e9f12226b146a66e6ec3c21a12a6c2d254

Reply:
Thanks!